CatchupArchiver/v1/normal.yaml

```diff
@@ -0,0 +1,5 @@
+settingsVersion: 2.0
+xmlVersion: 4.6
```

Comment:
I believe the current xmlVersion is 5.1.0, but it should be the version to which the CatchupArchiver was added.